paket.dependencies

```diff
@@ -1,5 +1,6 @@
 source https://nuget.org/api/v2
 
+nuget Nuget.CommandLine
```

Comment:
you need to pass the api key to the build.

I always have a (gitignored) release.cmd in my repos. You can see http://fsprojects.github.io/ProjectScaffold/release-process.html how it works.